packages/simple-hub/src/wallet/__test__/ledger-response.json

```diff
@@ -0,0 +1,55 @@
+[
```

Comment:
Can you auto-generate the test message data?

Reply:
Yep, we can. Probably worth doing down the line.

Reply:
581b0669